rfcs/2021-10-icons-in-core-DS.md

```diff
@@ -0,0 +1,102 @@
+# RFC: Standardizing Icons in the Core DS
+
+**Date**: October 01, 2021
+**Status**: `Review`
+
+## Problem
+
+The core DS does not have a consistent and easy way for consuming applications to import SVG icons. Additionally, because of the way some icons are used in DS components, it is difficult for consuming applications to change styles for an icon (like color, size, positioning, etc).
+
+### Background
+
+Currently, icons are stored in three main ways:
+
+1. Exported as an individual React component, but not in a centralized location
+2. Stored in the `design-system/images/` directory as .svg files
+3. Included in a component's markup unable to be re-used individually.
+
+The .svg files in the `images/` directory are referenced in the code in two main ways:
+
+1. In CSS as background images
+2. Using the library `inline-react-svg`, .svg files can be imported and used as a component (Note: this does not work with Typescript components)
+
+## Proposal
+
+Icons in the core DS should be moved to a centralized location and each icon should be standardized in size, accessibilty feature, and exported individually as a React component. This will provide more flexibility to consuming applications by allowing them to import individual SVG icons, adjust the styling on them with more ease, and create icon consistency among products.
```

Comment:
> ...should be standardized in size, accessibilty feature, and exported individually...

That second item in the list is a little confusing. Should it be "standardized in size and accessibility features and exported..."? Does the "standardized" apply to "accessibility features"?